app/serializers/service_serializer.rb

```diff
@@ -11,13 +11,13 @@ def serialize_service
       :recorded_by_educator_id,
       :recorded_at
     ]).merge({
-      discontinued_by_educator_id: discontinued.try(:recorded_by_educator_id),
+      discontinued_by_educator_id: discontinued.try(:discontinued_by_educator_id),
```

Comment:
@alexsoble I changed this to `discontinued_by_educator_id` as well.

Reply:
👍